src/components/Modal/index.styles.js

```diff
@@ -160,6 +160,7 @@ const contentGutters = css`
 export const ModalContent = styled.div`
   background-color: ${themes.global.white.base};
   overflow-y: auto;
+  line-height: ${typography.lineHeight.body};
```

Comment:
Looks good. For future reference, there is also a [mixin](https://github.com/ticketmaster/aurora/blob/master/src/mixins/lineHeight.js) that you can use 😄